beaker-vue/src/components/notebook/BeakerNotebookPanel.vue

```diff
@@ -75,8 +75,7 @@ function handleMoveCell(fromIndex, toIndex) {
  * As well as dataTransfer so that drop target knows which one was dropped
  **/
 function handleDragStart(event, beakerCell, index)  {
-    if (event.dataTransfer !== null) {
-
+    if (event.dataTransfer !== null && typeof event?.target?.closest !== "undefined") {
```

Comment:
2 things:
1) AFAIK, the preferred way to check if a JS/TS object is undefined is `if (item === undefined)` rather than `if (typeof item === "undefined"). That is doing a singleton entity comparison rather than a string comparison. My understanding is that it's faster and less prone to issue. I definitely could be wrong though. Do you have a particular reason for using the latter?

2) I just tested this and can still get it to trigger with this code. Taking a quick peek to see if there's a more fundamental something else we can do.